### PR TITLE
168 fix types and stub service

### PR DESCRIPTION
**Pull request by candleshine**
some budget record stubs need adding

### CODE REVIEW

File: src/components/budget/dto/create-budget.dto.ts [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
 import { MinLength, ValidateNested } from 'class-validator';
-import { Field, InputType, ObjectType } from 'type-graphql';
+import { Field, ID, InputType, ObjectType } from 'type-graphql';
 import { Budget, BudgetStatus } from '../budget';
 
 @InputType()
 export abstract class CreateBudget {
   @Field({ nullable: true })
   @MinLength(2)
   readonly status: BudgetStatus;
```
Comment (CarsonF) [inline src/components/budget/dto/create-budget.dto.ts]: I think status is just Pending to start with, no need to specify here

File: src/components/budget/dto/update-budget.dto.ts [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+import { Budget, BudgetRecord } from './budget';
 
 @InputType()
 export abstract class UpdateBudget {
   @Field(() => ID)
   readonly id: string;
 
   @Field()
-  readonly status: SecuredBudgetStatus;
+  readonly status: BudgetStatus;
```
Comment (CarsonF) [inline src/components/budget/dto/update-budget.dto.ts]:
```ts
@Field(() => BudgetStatus, { nullable: true })
readonly status?: BudgetStatus;
```

**Comment by candleshine** [comment]
need to fix resolver

File: src/components/budget/dto/create-budget.dto.ts (hunk 1 above)
Comment (candleshine) [reply src/components/budget/dto/create-budget.dto.ts]: that's fine

**Pull request merged**